### PR TITLE
Add option to configure metrics export option when scheduling job

### PR DESCRIPTION
**Pull request by arjansh**
Add the option to active metrics export when scheduling jobs from DataCleaner desktop.

### CODE REVIEW

File: monitor/api/src/main/java/org/datacleaner/monitor/scheduling/model/ScheduleDefinitionModel.java [hunk 1]
```diff
@@ -24,6 +24,9 @@
 
 public class ScheduleDefinitionModel {
     private final String _hotFolder;
+    
+    @JsonProperty("exportMetrics")
+    private boolean _exportMetrics;
 
     @JsonCreator
     public ScheduleDefinitionModel(@JsonProperty("hotFolder") final String hotFolder) {
```
Comment (LosD) [inline monitor/api/src/main/java/org/datacleaner/monitor/scheduling/model/ScheduleDefinitionModel.java]: Maybe we should also move this `@JsonProperty` to a setter? I'd prefer this an immutable object, but as long as we're consistent, both is fine with me.

**Comment by LosD** [comment]
Single comment on consistency, otherwise LGTM.

I don't think there's anything to test.

File: monitor/api/src/main/java/org/datacleaner/monitor/scheduling/model/ScheduleDefinitionModel.java (hunk 1 above)
Comment (arjansh) [inline monitor/api/src/main/java/org/datacleaner/monitor/scheduling/model/ScheduleDefinitionModel.java]: I was considering the immutability too, but I'm afraid we may want to add more options later and that will result in a constructor with a lot of arguments. Just say it, if you'd rather have it immutable, I'll change it.

Comment (LosD) [inline monitor/api/src/main/java/org/datacleaner/monitor/scheduling/model/ScheduleDefinitionModel.java]: No, it's fine. But make them consistent so they're both using setters, then.

**Comment by ClaudiaPHI** [comment]
LGTM!

**Pull request merged**